### DISCUSSION TITLE
Does Speculation Rules Work with Livewire?

### DISCUSSION THREAD

**Discussion by kornatzky** [InnoGE/filament-form-faker#9] [Q&A] [answered]
If you have pages that render Livewire components, will this speed it up?

**Comment by geisi** [comment, marked as answer]
@kornatzky 
You have asked your question on the wrong repository this would be the correct one:
https://github.com/InnoGE/laravel-speculation-rules-api

Short answer: yes.

Longer answer: In the Livewire context, you could use `wire:navigate` with the `.hover` setting. This will do something similar with better browser compatibility. If you have problems with `wire:navigate,` then the speculation rules API is a nice alternative.

**Reply by kornatzky** [reply]
Thanks
